process-services/1.11/install/container.md

```diff
@@ -0,0 +1,302 @@
+```
+
+```bash
+docker run -p {port}:8080 alfresco/process-services-admin
+```
+
+For example, to run Process Services Administrator on port 8095 use the following:
+
+```bash
+docker run -p 8095:8080 alfresco/process-services-admin
+```
+
+Once the containers have started up, visit the following URLs to access the applications:
+
+* `http://localhost:{port}/activiti-app`
+* `http://localhost:{port}/activiti-admin`
+
+For example `http://localhost:8095/activiti-admin` if running the Administrator application on port 8095.
+
+>**Note:** Docker for Desktop is not a production environment.
+
+It is possible to override the default environment variables for Process Services and Process Services Administrator,
+see next sections.
+
+### Configure Process Services variables
+
+It is possible to override the default variable values used by the Docker container.
+
+There are three options for specifying your own variables during a Docker deployment:
+
+* Mount your own `activiti-app.properties` and optionally an `activiti-identity-service.properties` file in `/usr/local/tomcat/lib` using Docker volumes.
+* Specifying environment variables for each properties file that points to an accessible location such as an S3 bucket:
+
+    * Use the `EXTERNAL_ACTIVITI_APP_PROPERTIES_FILE` environment variable for an `activiti-app.properties` file.
+    * Use the `EXTERNAL_ACTIVITI_IDENTITY_SERVICE_PROPERTIES_FILE` environment variable for an `activiti-identity-service.properties` file.
+        >**Note:** If you choose this option, the files will be automatically downloaded into the contextual folder.
+* Configure the environment variables in the Docker container by overriding the default values.
+
+Variables that correspond to the `activiti-app.properties` file:
+
+|Property|Description|
+|--------|-----------|
+|ACTIVITI_DATASOURCE_DRIVER|The JDBC driver used to connect to the database. The default is `org.h2.Driver`. |
+|ACTIVITI_HIBERNATE_DIALECT|The dialect that Hibernate uses that is specific to the database type. The default is `org.hibernate.dialect.H2Dialect`. |
+|ACTIVITI_LICENSE_MULTI_TENANT|Set whether the license used is a multi-tenant one or not. The default is `false`. |
+|ACTIVITI_DATASOURCE_URL|The location of the database that will be used. The default is `jdbc:h2:mem:db1;DB_CLOSE_DELAY=1000`. |
+|ACTIVITI_DATASOURCE_USERNAME|The username to access the database with. The default is `alfresco`. |
+|ACTIVITI_DATASOURCE_PASSWORD|The password for the `ACTIVITI_DATASOURCE_USERNAME` user. The default is `alfresco`. |
+|ACTIVITI_ADMIN_EMAIL|The email address for the default administrator user. The default is `admin@app.activiti.com`. |
+|ACTIVITI_ADMIN_PASSWORD_HASH|The hashed password for `ACTIVITI_ADMIN_EMAIL` user. The default is ``. |
+|ACTIVITI_CORS_ENABLED|Sets whether Cross Origin Resource Sharing (CORS) is enabled or not. The default is `true`. |
+|ACTIVITI_CORS_ALLOWED_ORIGINS|The host origins allowed in CORS requests. The default is `*`. |
```

Comment:
`*` can no longer used as the value for this property. By default it's not set. A new property `ACTIVITI_CORS_ALLOWED_ORIGIN_PATTERNS` should be used in case `*` needs to be used.

Reply:
This change is for 2.x.

The default is not set and cannot be * ever.

The description will be the host origins patters allowed in CORS requests. the default is * but can also be a pattern.

Reply:
Cheers
added: 
|ACTIVITI_CORS_ALLOWED_ORIGIN_PATTERNS| The host origins patterns allowed in CORS requests. The default is `*` but you can also use a pattern.|

Reply:
Added/adjusted to 

|ACTIVITI_CORS_ALLOWED_ORIGINS|The host origins allowed in CORS requests. There is not a default value set. You can't use `*`.|
|ACTIVITI_CORS_ALLOWED_ORIGIN_PATTERNS| The host origin patterns allowed in CORS requests. The default is `*` but you can also use a pattern.|